php.ini

```diff
@@ -0,0 +1 @@
+date.timezone = UTC
```

Comment:
Could you create this file right from the Dockerfile?

Reply:
Yes definitely, want me to change it to use that?

Reply:
OK just commited and pushed